static/journal.js

```diff
@@ -0,0 +1,56 @@
+
+$( document ).ready( function() {
+  $("#edit_form").hide();
+
+  function add_post() {
+    $.ajax({
+      url: "/add",
+      type: "POST",
+      dataType: 'json',
+      data: { "title": $("#title").val(), 'text': $("#text").val()},
+      success: success
```

Comment:
I don't see a `success` method anywhere in your javascript here.  Where is it defined?